.github/ISSUE_TEMPLATE/BugReporting.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
```

Comment:
@Jayakumar03 , The File name should be `bug.yml`.

Reply:
Done